configure.ac

```diff
@@ -1859,6 +1859,9 @@ AS_IF([test "x$enable_experimental_plugins" = "xyes"],
 AM_CONDITIONAL([BUILD_REMAP_STATS_PLUGIN],
   [ test "x$enable_experimental_plugins" = "xyes" -a "x$ac_cv_header_search_h" = "xyes" -a "x$ac_cv_type_struct_hsearch_data" = "xyes" -a "x$ac_cv_func_hcreate_r" = "xyes" -a "x$ac_cv_func_hsearch_r" = "xyes" ])
 
+AC_ARG_VAR(TS_DEFAULT_STACK_SIZE, [The default stack size for threads in bytes])
+TS_DEFAULT_STACK_SIZE="${TS_DEFAULT_STACK_SIZE:-1048576}"
```

Comment:
Hmmm, this is odd. You want to modify the stack size via an environment variable to configure? Wouldn't it be better to add a --with-stack-size=4MB option or some such ?